### PR TITLE
Release 3

### PR DESCRIPTION
**Pull request by ju-cu**
Upgrade to Java 17 and Curity Identity Server 7.3.0
Replace deprecated method
Bump version to 3.0.0

### CODE REVIEW

**Review by mtrojanowski — APPROVED** [review]

**Review by iggbom — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
```
Comment (mtrojanowski) [inline LICENSE:189]: I was wondering whether we should have 2022 here, or the year when the plugin was first created (2020). I think that was the start of the copyright.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
 # Time Based Deny Authentication Action
 
-[![Build Status](https://travis-ci.org/curityio/time-authentication-action.svg?branch=master)](https://travis-ci.org/curityio/time-authentication-action)
+[![Build Status](https://travis-ci.com/curityio/time-authentication-action.svg?branch=master)](https://travis-ci.com/curityio/time-authentication-action)
```
Comment (mtrojanowski) [inline README.md]: Do we have an account on travis-ci.com? Currently, it looks like the time-authentication-action project does not exist there? Or should I have some permissions to access it?

File: README.md [hunk 3]
```diff
@@ -48,4 +48,4 @@ This plugin and its associated documentation is listed under the [Apache 2 licen
 
 Please visit [curity.io](https://curity.io/) for more information about the Curity Identity Server.
 
-Copyright (C) 2020 Curity AB.
```
Comment (mtrojanowski) [inline README.md]: I think we should leave 2020, as that's when the plugin was first published, right?

File: README.md (hunk 2 above)
Comment (ju-cu) [reply README.md]: True. I didn't double check that the project doesn't exist on `travis-ci.com`. I assumed it was moved. It exists here: https://travis-ci.org/github/curityio/time-authentication-action but there's a note: 
> [...] the building on [travis-ci.org](http://www.travis-ci.org/) is ceased. Please use [travis-ci.com](http://www.travis-ci.com/) from now on.

For the purpose of this PR, I think, we can remove the build status from the Readme. We don't usually specify that.

File: README.md (hunk 3 above)
Comment (ju-cu) [reply README.md:51]: Agree, we should keep the Copyright but I'd update it to 2022.

File: LICENSE (hunk 1 above)
Comment (ju-cu) [reply LICENSE:189]: We should keep Copyright and update it with the correct values instead, i.e. "Copyright 2022 Curity AB"

**Review by mtrojanowski — APPROVED** [review]

**Pull request merged**